lib/archethic/transaction_chain.ex

```diff
@@ -793,16 +793,39 @@ defmodule Archethic.TransactionChain do
   end
 
   defp do_fetch_transaction_chain(nodes, address, paging_state, opts \\ []) do
+    order = Keyword.get(opts, :order, :asc)
+
     conflict_resolver = fn results ->
       results
       |> Enum.sort(
-        &((&1.more? and !&2.more?) or length(&1.transactions) > length(&2.transactions))
+        # Prioritize more? at true
+        # then length of transaction list
+        # then regarding order, the oldest or newest transaction timestamp
+        # of the first element of the list
+        &with false <- &1.more? and !&2.more?,
```

Comment:
What happens with the `else` clause ? Any catch ?

Reply:
The else clause will return true and will be catched by the sort function and apply the sort

Reply:
Indeed the result of the with is the opposite of the clauses, so true